astropy/units/quantity.py

```diff
@@ -7,6 +7,8 @@
 
 from __future__ import annotations
 
+import builtins
+
 # STDLIB
```

Comment:
This is yet another example of the `# STDLIB` comment actively preventing Ruff from keeping the imports organized. We really should get rid of such comments in all of `astropy`.

Reply:
I agree with the sentiment - so, @Deen-dot, if you can get rid of those that certainly is fine!